midend/removeParameters.h

```diff
@@ -83,28 +86,42 @@ class FindActionParameters : public Inspector {
     void postorder(const IR::MethodCallExpression* expression) override;
 };
 
-// Removes parameters of an action which are in/inout/out.
-// For this to work each action must have a single caller.
-// (This is done by the LocalizeActions pass).
-// control c(inout bit<32> x) {
-//    action a(in bit<32> arg) { x = arg; }
-//    table t() { actions = { a(10); } }
-//    apply { ... } }
-// is converted to
-// control c(inout bit<32> x) {
-//    bit<32> arg;
-//    action a() { arg = 10; x = arg; }
-//    table t() { actions = { a; } }
-//    apply { ... } }
+/**
+Removes parameters of an action which are in/inout/out.
```

Comment:
Your commenting style is inconsistent, sometimes you use a star, sometimes you just indent, and sometimes you don't indent.